packages/map/src/components/UsaMap/components/UsaMap.State.tsx

```diff
@@ -118,6 +119,54 @@ const UsaMap = () => {
     }
   }, [data, state.general.territoriesAlwaysShow])
 
+  // useEffect(() => {
```

Comment:
what's the reason for keeping this?

Reply:
I removed it on accident, it needs to stay in to keep the pattern contrast checks in place.